extensions/indexes/lucene/src/main/java/org/exist/xquery/modules/lucene/Query.java

```diff
@@ -197,16 +197,17 @@ public int getOptimizeAxis() {
     }
 
     public NodeSet preSelect(Sequence contextSequence, boolean useContext) throws XPathException {
-    	if (contextSequence != null && !contextSequence.isPersistentSet())
+        // DW: contextSequence can be null
```

Comment:
my name can be removed :-)

Reply:
Well, it was your comment that led me to the solution :)